tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "outDir": "./dist",
     "sourceMap": true,
     "strict": false,
-    "target": "ES6"
+    // "target": "ES6",
+    "target": "ES5"
```

Comment:
what is the size difference between ES5 and 6 (which I believe was smaller) ? is that required for IE. need to trade off all browser vs obsolete IE